jsonnet-mode.el

```diff
@@ -46,6 +46,12 @@
   :type '(string)
   :group 'jsonnet)
 
+(defcustom jsonnet-lib-dirs
```

Comment:
nit: name this jsonnet-library-search-directories

Reply:
I like it. Please see 3546f107511f551363eb755b8d35ef1da2455d83.